modules/grizzly/src/main/java/org/glassfish/grizzly/Connection.java

```diff
@@ -492,4 +492,10 @@ void setProcessorSelector(
         LOCALLY, REMOTELY
     }
 
+    default ClassLoader getLoggerClassLoader(){
```

Comment:
`logger` is just how it will be used in mule, but it has other implicancies. Rename accordingly